pkg/encoding/v1/encoding.go

```diff
@@ -319,6 +320,9 @@ func (d *Decoder) Decode(data []byte) (*object.OpenCompose, error) {
 			Name: string(s.Name),
 		}
 
+		// user given value of replicas
```

Comment:
it's now a pointer to the value; maybe just drop the comment since its obvious anyway